apm-agent-plugins/apm-kafka-plugin/apm-kafka-headers-plugin/pom.xml

```diff
@@ -27,5 +27,10 @@
             <artifactId>apm-kafka-base-plugin</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.ivy</groupId>
+            <artifactId>ivy</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
We might find a better solution to avoid duplication of common plugin test dependencies, but it would be better handled in a separate cleanup PR.